bahati10/coa-taskforce-photo-gallery-challenge
# FEATURE: added jJS intection functionalities and styles

**What does this PR do?**
This PR implements an interactive photo gallery that displays additional information about each image on hover and includes a "know more" button for further actions.

### Description of Task to be completed?
Create the main container and individual gallery items in HTML.
Style the gallery and its items using CSS to ensure a responsive and visually appealing layout.
Add hover effects to display additional information overlays using CSS transitions.
Implement JavaScript functionality to handle hover events and dynamically change styles to show/hide elements and apply effects.
### How should this be manually tested?

1. Open the HTML file in a web browser.
2. Verify that the gallery items are displayed correctly.
3. Hover over each gallery item to ensure that the image applies a grayscale filter, blurs, and scales up slightly.
4. Check that the information overlay (species name, place, and "know more" button) becomes visible on hover.
5. Ensure that the effects revert when the hover state ends.
### Any background context you want to provide?
N/A
### Screenshots (if appropriate)
N/A
### Questions:
N/A
